### DISCUSSION TITLE
Why there is no try-catch inside `isMinimumWebViewInstalled`? (May be a bug)

### DISCUSSION THREAD

**Discussion by amitsingh19975** [ionic-team/capacitor#6793] [Q&A]
Recently, I got a crash. referring to `int majorVersion = Integer.parseInt(majorVersionStr);`. Where the major version string is found to be 'Developer Build' for the device 'Huawei Enjoy 9s'. Is this intentional or a bug?

[isMinimumWebViewInstalled](https://github.com/ionic-team/capacitor/blob/0cc927ef5f0f7076a6d486d666d78483f1d71c54/android/capacitor/src/main/java/com/getcapacitor/Bridge.java#L299)

**Comment by chrismclarke** [comment]
Think this is fixed with #6907

**Comment by amitsingh19975** [comment]
Okay, thanks. I did my workaround by wrapping in a try-catch. From the looks of it, I can move to the latest fix soon.
